src/datajudge/requirements.py

```diff
@@ -270,17 +275,30 @@ def add_uniques_equality_constraint(
         of a ``DataSource`` are strictly the ones of a reference set of expected values,
         specified via the ``uniques`` parameter.
 
+        Null values in the column are ignored. To assert the non-existence of them use
+        the ``NullAbsence`` constraint via the ``add_null_absence_constraint`` helper method
```

Comment:
Afaict there isn't really a `NullAbsence` constraint but `add_null_absence_constraint` creates a `MaxNullFraction constraint.

```suggestion
        the :meth:`~datajudge.requirements.WithinRequirement.add_null_absence_constraint`` helper method
```

Reply:
Updated.